package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "dotcomchrome",
+  "version": "2.0.0",
+  "description": "Header and footer templates from mbta.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mbta/dotcomchrome.git"
```

```suggestion
    "url": "https://github.com/mbta/dotcomchrome.git"
```

I _think_ the proper notation omits the `git+`